code/game/rollcredits.dm

```diff
@@ -0,0 +1,184 @@
+	if(SNOW_THEME)
+		possible_titles += "A VERY [pick("NANOTRASEN", "EXPEDITIONARY", "DIONA", "PLASMA", "MARTIAN")] CHRISTMAS"
+	possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "MUCH ADO ABOUT NOTHING", "WHAT HAPPENS WHEN YOU MIX MOMMIS AND COMMERCIAL-GRADE PACKING FOAM", "ATTACK! ATTACK! ATTACK!", "SEX BOMB", "THE BALLAD OF [uppertext(station_name())]")]"
+	possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELIQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
+	possible_titles += "[pick("THE DAY [uppertext(station_name())] STOOD STILL", "HUNT FOR THE GREEN WEENIE", "ALIEN VS VENDOMAT", "SPACE TRACK")]"
+	titles += "<center><h1>SEASON [rand(1,22)] EPISODE [rand(1,17)]<br>[pick(possible_titles)]<h1></h1></h1></center>"
+	titles += "<center><h1>CAST:</h1><h2>(in order of appearance)"
+	for(var/mob/living/carbon/human/H in living_mob_list|dead_mob_list)
+		if(!star || H.talkcount > star.talkcount)
+			star = H
+		if(H.timeofdeath && H.timeofdeath < 5 MINUTES) //don't mention these losers (prespawned corpses mostly)
+			continue
+
+		chunk += gender_credits(H)
+		chunksize++
+		if(chunksize == 5)
+			cast += chunk
+			chunk = "<center>"
+			chunksize = 0
+	titles += cast
+	if(chunksize)
+		titles += chunk
+
+	var/list/corpses = list()
+	for(var/mob/living/carbon/human/H in dead_mob_list)
+		if(H.timeofdeath < 5 MINUTES) //no prespawned corpses
+			continue
+		else if(H.real_name)
+			corpses += H.real_name
+	if(corpses.len)
+		titles += "<center>BASED ON REAL EVENTS<br>In memory of [english_list(corpses)].</center>"
+
+	var/list/staff = list("<h1>PRODUCTION STAFF</h1>")
+	var/list/staffjobs = list("Coffee Fetcher", "Cameraman", "Angry Yeller", "Chair Operator", "Choreographer", "Historical Consultant", "Costume Designer", "Chief Editor", "Executive Assistant")
+	for(var/client/C)
+		if(!C.holder)
+			continue
+		if(C.holder.rights & (R_DEBUG|R_ADMIN))
+			var/observername = ""
+			if(C.mob && istype(C.mob,/mob/dead/observer))
+				var/mob/dead/observer/O = C.mob
+				if(O.started_as_observer)
+					observername = "[O.real_name] a.k.a. "
+			staff += "<h2>[uppertext(pick(staffjobs))] - [observername]'[C.key]'</h2>"
+
+	var/disclaimer = "Sponsored by The United States Navy.<br>All rights reserved.<br>"
+	disclaimer += pick("All stunts were performed by underpaid and expendable interns. Do NOT try at home.", "Nanotrasen does not endorse behaviour depicted. Attempt at your own risk.<br>")
+	titles += "<center>[disclaimer]</center>"
+	titles += "<center>This motion picture is protected under the copyright laws of the United States and other countries throughout the world. Country of first publication: United States of America. Any unauthorized exhibition, distribution, or copying of this film or any part thereof (including soundtrack) may result in civil liability</center>"
```

Comment:
No it's not

Reply:
https://en.wikipedia.org/wiki/All_persons_fictitious_disclaimer

Reply:
It's not protected under the copyright laws of the united states. This is stuff that can get weird if somebody decides to chase up on it